test/runnable/testassert.d

```diff
@@ -96,10 +101,12 @@ void test20375() @safe
         this(this) @safe
         {
             instances++;
+            postblits++;
         }
 
         ~this() @safe
         {
+            printf("dtor\n"); // make the dtor non-nothrow
```

Comment:
You could replace the `printf`s with something like this (tested with Windows x64 host -> x32 executable)

```suggestion
           // make the dtor non-nothrow
            if (postblits > 100)
                throw new Exception("");
```

We currently do not check the executables output unless a  `RUN_OUTPUT` section is present but that should change once all missing sections were added (similarly to how it was done with `TEST_OUTPUT`).

Reply:
Yep, done.